File: frontend/src/app/models/response/contact-response.ts
import { CompanyResponse } from "./company-response";

export interface ContactResponse {
    id: string;
    fullName: string;
    imageUrl: string;
    email: string;
    title: string;
    phone: string;
    isCustomer: boolean;
    companyResponse: CompanyResponse;
    createdBy: string;
    updatedBy: string;
    createdAt: string;
    updatedAt: string;
}

File: backend/src/main/java/org/gestionstock/stock/Payload/Response/ContactResponse.java
package org.gestionstock.stock.Payload.Response;

public record ContactResponse(
    String id,
    String fullName,
    String imageUrl,
    String email,
    String title,
    String phone,
    boolean isCustomer,
    CompanyResponse companyResponse,
    String createdBy,
    String updatedBy,
    String createdAt,
    String updatedAt
){
    
}
